save-common/src/jvmMain/kotlin/org/cqfn/save/entities/AgentStatus.kt

```diff
@@ -0,0 +1,15 @@
+package org.cqfn.save.entities
+
+import org.cqfn.save.agent.AgentState
+import java.time.LocalDate
+import javax.persistence.*
+
+@Entity
+class AgentStatus (
+        @Column(name="time")
+        var time: LocalDate,
+        @Enumerated(EnumType.STRING)
+        var state: AgentState,
+        var agentId: Long,
```

Comment:
agentId should be unique: https://docs.jboss.org/hibernate/jpa/2.1/api/javax/persistence/UniqueConstraint.html